src/app/onion/core/learning-object.service.ts

```diff
@@ -376,28 +376,6 @@ export class LearningObjectService {
       .toPromise();
   }
 
-  /**
-   * Unsubmit a learning object
-   *
-   * @param {learningObject} learningObject the learning object to be unpublished
-   */
-  unsubmit(learningObject: LearningObject) {
```

Comment:
the cancelSubmission function in the dashboard component uses this function. Is there a reason for deleting it?

Reply:
This `unsubmit` function exists in both `submission.service.ts` and `learning-object.service.ts` files, doing the same thing in both files. So that's why I removed it from the `learning-object.service.ts`.

The `cancelSubmission` function uses the `unsubmit` function from the `submission.service.ts`.